packages/ui/src/components/ChatMessage/ChatMessage.tsx

```diff
@@ -117,6 +117,26 @@ const ChatMessage = ({
     return utils.makePrettyTime(date);
   }, [post.sentAt]);
 
+  const messageInputForEditing = useMemo(
+    () => (
+      <MessageInput
+        groupMembers={[]}
+        storeDraft={() => {}}
+        clearDraft={() => {}}
+        getDraft={async () => ({})}
+        shouldBlur={false}
+        setShouldBlur={() => {}}
+        send={async () => {}}
```

Comment:
I started going through `MessageInput` and marking these all as optional, but it seemed a little dicey – I think we should consider this usage a hack and eventually make a unique "edit message" view, instead of adding branching logic to `MessageInput`